app/workers/sentry_job.rb

```diff
@@ -3,9 +3,18 @@
 class SentryJob
   include Sidekiq::Worker
 
-  sidekiq_options queue: 'tasker'
+  sidekiq_options queue: 'tasker', retry: false
 
   def perform(event)
     Raven.send_event(event)
+  rescue StandardError => e
+    Rails.logger.error(
+      "Error performing SentryJob: #{e.message}",
+      original_event: {
+        culprit: event&.culprit,
```

Comment:
I had the "opportunity" to observe one of these today and noticed in the logs:

`{"host":"xxx","application":"vets-api-worker","timestamp":"2018-10-03T22:09:43.366651Z","level":"warn","level_index":3,"pid":3658,"thread":"96883340","name":"Sidekiq","message":"NoMethodError: undefined method `culprit' for #<Hash:0x000000000e233da0>"}`

Reply:
@wyattwalter 😐if this PR isn't merged yet is that an error from within Raven? In this PR for safety's sake we could just log a simpler message.

Reply:
It's not merged, but I pushed this one out to a worker as a test to see if it would resolve the issue.

Reply:
Been following this issue in passing, so I could be missing something, but...
Sentry (Raven) events are (should) always just be [Ruby hashes](https://docs.sentry.io/clients/ruby/config/#optional-settings) in Raven 2+, which means it'll never respond to `culprit`.
Should it look like this?
```
original_event: {
        culprit: event[:culprit],
        extra: event[:extra],
        backtrace: event[:backtrace]
}
```
BTW, it looks like [`culprit` is deprecated](https://docs.sentry.io/clientdev/attributes/)